extensions/common/context.h

```diff
@@ -76,6 +76,14 @@ struct RequestInfo {
   // Response total size in bytes, include header, body, and trailer.
   int64_t response_size = 0;
 
+  // Total Request total size in bytes for TCP, as seen on the upstream
+  // connection.
+  int64_t tcp_request_size = 0;
```

Comment:
Maybe just the same field and call "Stream request" where stream is either an http1.1/h2 request body or a TCP connection.

Reply:
shouldn't we have rather total_request_size => 
http => body + header + trailer
tcp => connection data size
?

Reply:
Done.. just reusing the fields as using the values set in streaminfo for tcp for now...